src/style.css

```diff
@@ -0,0 +1,24 @@
+body {
+  font-size: 13px;
+  font-family: Arial;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin: 0;
+}
+
+h2 {
+  font-size: 15px;
+  font-weight: normal;
+  margin: 1em 0;
+}
+
+a {
+  color: green;
+}
+
+.hidden {
+  display: none;
+}
```

Comment:
you don't need class for this, just write dosplay: none where it's needed

Reply:
> you don't need class for this, just write dosplay: none where it's needed

I changed tag name in css on class, totally agree with with comment but I don't agree with comment about inline style in html. At first it is a bad practice for work, and second -  run task don't pass it. Maybe, I understand something wrong  - correct me.

Reply:
Mb you right about inline, but i don't see sense of hiding something in this task, in description of task nothing about it

Reply:
ok, so  I didn't understand the tаsk right  at first. I thought hidden a content after article 16 was a part of task (otherwise why is this content there)